core/invoker/src/main/scala/whisk/core/container/AkkaHttpUtils.scala

```diff
@@ -24,24 +24,38 @@ import scala.concurrent.duration.DurationInt
 import scala.concurrent.duration.FiniteDuration
 import scala.util.Try
 import akka.actor.ActorSystem
-import akka.http.scaladsl._
 import akka.http.scaladsl.model._
 import akka.stream.ActorMaterializer
-import akka.stream.scaladsl._
 import java.util.concurrent.TimeoutException
 
+import play.api.http.{ContentTypeOf, Writeable}
```

Comment:
we're not using this utility in the invoker - instead we use the apache http client.
@markusthoemmes is this likely to change with new invoker work?
perhaps we should move this utility to the test project.